modules/search/instant-search/instant-search.scss

```diff
@@ -32,3 +32,8 @@ $grid-size-large: 16px;
 .jetpack-search-filters-widget__filter-list {
 	list-style-type: none;
 }
+
+// Hide the search modal in the TwentyTwenty theme
+.theme-twentytwenty .cover-modal.show-modal.search-modal {
```

Comment:
We originally tried preventing this with JS, but targeted CSS works well and prevents console errors :)